test/Microsoft.Health.Fhir.Shared.Tests.Integration/Persistence/SqlServerSchemaUpgradeTests.cs

```diff
@@ -97,6 +97,17 @@ public async Task GivenASchemaVersion_WhenApplyingDiffTwice_ShouldSucceed(int sc
             }
         }
 
+        public static IEnumerable<object[]> GetSchemaVersions()
+        {
+            foreach (object item in Enum.GetValues(typeof(SchemaVersion)))
+            {
+                if ((int)item >= 7)
```

Comment:
I also like 7, where is this coming from? Is there a system const already defined somewhere?

Reply:
:) I just re-wrote the existing tests which started from version 7. The reason is that our schema upgrade scripts before v7 aren't idempotent. So we only run this test for v7 onwards.

Reply:
Added comment to the test to explain this